types.d.ts

```diff
@@ -11839,6 +11994,7 @@ declare namespace exports {
 			export { HttpUriPlugin, HttpsUriPlugin };
 		}
 	}
+	export type LoaderDefinition = LoaderDefinition<EmptyContextAdditions>;
```

Comment:
Hopefully these changes will lead to `export { LoaderDefintion }` being generated...

But maybe there is also a bug in the type generation. We didn't export generic types yet...